content/docs/ui/managing-contacts/email-address-validation.md

```diff
@@ -14,23 +14,23 @@ seo:
 ---
 <call-out type="warning">
 
-Email Validation is currently in beta. During beta, any participants with Pro or Premier accounts will have access to this feature. Functionality and performance is subject to change while in beta.
+Email Validation API is currently in beta. Customers with Pro or Premier Email API plans will have access to this feature free during Beta, once generally available [this pricing](https://sendgrid-email-validation.api-docs.io/v3/getting-started/authentication) will apply. Functionality and performance is subject to change while in beta.
 
 </call-out>
 
-Email Address Validation provides you real-time detailed information on the validity of your users' email addresses, as well as allows you to integrate this validation process into your platform's signup form. You can customize the best use of email address validation for your use case. You can identify a validation threshold from which you feel comfortable sending a follow up email, display to your users that their email addresses are likely invalid within your form, and ultimately decide if you want to drop invalid email address from your database.
+Email Address Validation provides real-time detailed information on the validity of email addresses. You can integrate this validation process into your platform's signup form and customize the best use of email address validation for your use case. You can identify a validation threshold from which you feel comfortable sending a follow up email, display to your form visitors that their email addresses are likely invalid within your form, and ultimately decide if you want to drop invalid email addresses from your database.
```

Comment:
These read like a list of potential use cases to me - would bullets read better? 

You can integrate this validation process into your platform's signup form and customize the best use of email address validation for your use case: 
- Identify a validation threshold from which you feel comfortable sending a follow up email 
- Display to visitors that their email addresses are likely invalid within your form, prompting a suggestion to fix
- Decide if you want to drop invalid email addresses from your database.

Reply:
updated, but I don't think I used the right bullets for markup?